tests/python/contrib/test_arm_compute_lib/test_dense.py

```diff
@@ -251,7 +252,14 @@ def test_codegen_dense():
         verify_codegen(func, exp_codegen)
 
 
-def test_qnn_dense():
+@pytest.mark.parametrize(
+    "dtype,min_range,max_range",
+    [
+        ("uint8", 0, 255),
+        ("int8", -127, 128),
+    ],
+)
```

Comment:
For future usage, could we define a function either with acl tests / under tvm testing that would provide the ranges?
Similar to https://github.com/apache/tvm/blob/2105b937cd6b4f426156813d6562f2845e5604f0/tests/python/contrib/test_cmsisnn/utils.py#L89

Reply:
That's a good idea, I can start moving that direction when implementing the next operators.

Reply:
+1, I intended to do this for NPU testing as well but never got around to it, perhaps we can pull it out into a bigger refactoring?

Reply:
Yes, moving this into TVM testing area would be more beneficial.